vendor/xssbot/dockerfiles/Dockerfile.chrome

```diff
@@ -0,0 +1,38 @@
+FROM node:16-stretch-slim as base
```

Comment:
should we use buster instead?

Reply:
Oh yeah probably, i copy pasted this from last year hehe